### PR TITLE
Couple of documentation changes & background fix

### PR DESCRIPTION
**Pull request by ghost**
Added my instance to the list, and added a sample configuration file for Apache.

### CODE REVIEW

**Comment by zoski** [comment]
Hi, I notice you enabled rewriteEngine in your vhost. Why ? Is it usefull here ?

**Comment by Komic** [comment]
Hi @zoski, I can confirm it's not useful, I use it for a totally unrelated purpose and forgot to remove it from the sample.

File: docs/Running-Mastodon/Production-guide.md [hunk 1]
```diff
@@ -82,6 +82,47 @@ server {
...
+
+<VirtualHost *:443>
+   ServerAdmin contact@example.com
+   ServerName example.com
+
+   DocumentRoot /home/mastodon/live/public/
+
+   Header add Strict-Transport-Security "max-age=31536000"
+   SSLEngine on
+   SSLProtocol All -SSLv2 -SSLv3
```
Comment (Angristan) [inline docs/Running-Mastodon/Production-guide.md]: `SSLProtocol -all +TLSv1.2` is better

File: docs/Running-Mastodon/Production-guide.md [hunk 2]
```diff
@@ -82,6 +82,47 @@ server {
...
+   SSLHonorCipherOrder on
+   SSLCipherSuite EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH:AES128+EECDH:AES128+EDH
```
Comment (Angristan) [inline docs/Running-Mastodon/Production-guide.md]: You shoud disabled DHE ciphers : 
`SSLCipherSuite EECDH+AESGCM:AES256+EECDH:AES128+EECDH`

**Review by angristan — CHANGES_REQUESTED** [review]

File: docs/Running-Mastodon/Production-guide.md (hunk 2 above)
Comment (Komic) [inline docs/Running-Mastodon/Production-guide.md]: Heh, imo it's a better security/compatibility compromise but alright, I'll push that later.

**Comment by Komic** [comment]
Done, @Angristan (sorry, this PR is a bloody mess).

**Comment by Gargron** [comment]
We are moving documentation into [its own repository](https://github.com/tootsuite/documentation), so could you please reapply this PR there? Thank you for your time and effort and sorry for the inconvenience!

**Comment by Komic** [comment]
https://github.com/tootsuite/mastodon/pull/1560 https://github.com/tootsuite/documentation/pull/6 done~

**Pull request closed without merging**